cocos/scene-graph/component-scheduler.ts

```diff
@@ -84,15 +85,35 @@ function stableRemoveInactive (iterator, flagToClear): void {
     }
 }
 
+type InvokeFunc = (...args: unknown[]) => void;
+
 // This class contains some queues used to invoke life-cycle methods by script execution order
 export class LifeCycleInvoker {
     public static stableRemoveInactive = stableRemoveInactive;
 
+    /**
+     * @engineInternal please access `_zero` instead.
+     */
+    public get _internal_zero (): js.array.MutableForwardIterator<any> {
```

Comment:
If suggest to assess property instead, than why add this function?

Reply:
because _zero is a protected property，and NodeActivator cannot access it

Reply:
1. The comment is confused.
2. The property name is not good, why just use `zero` for property name?

Reply:
here is the comment for zero neg pos property
https://github.com/cocos/cocos-engine/blob/2a5b5711c1692c235bf05275b73cabacc9b15503/cocos/scene-graph/component-scheduler.ts#L91C58-L102

I'll optimize the comment for `_internal_zero _internal_neg _internal_pos`

Reply:
updated here https://github.com/cocos/cocos-engine/pull/15444/commits/881b44dcd45ef8b510b7ae2e31a9d8d206b15356

Reply:
`_internal_zero` is not a good name:
- we don't use `_` to connect words
- why not just use `zero`?

Reply:
though it's a public property, but I prefer to use `_` at the beginning of the name, so that even if the developers debug at runtime, they can see that this is a js-style private interface

can we just name it as `_internalZero` ?

Reply:
I've try updating here 
https://github.com/cocos/cocos-engine/pull/15444/commits/06f480c7be8f44e799d76cf2fe15500cf8b69934

Reply:
Why can not just use `zero`? I think it is more suitable in script. As it is engine internal, so i don't think need `_` prefix.

Reply:
update https://github.com/cocos/cocos-engine/pull/15444/commits/1c0cd24d40c4d1928afedad34dfe7edc4aee9faa